core/pom.xml

```diff
@@ -71,12 +71,12 @@
       <scope>compile</scope>
       <!--Dependency added by RoostGPT-->
     </dependency>
-    <dependency>
+    <!-- <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-context-annotation</artifactId>
       <version>5.2.5.RELEASE</version>
```

Comment:
spring-context-annotation dependency already has features of spring-beans which is already present in the Pom file, so this can be removed